Llk/Sampler/Sampler.php

```diff
@@ -190,9 +221,13 @@ protected function generateToken(Compiler\Llk\Rule\Token $token)
         $toNamespace = $this->completeToken($token);
         $this->setCurrentNamespace($toNamespace);
 
-        return $this->_tokenSampler->visit(
-            $token->getAST()
-        ) . ' '; // use skip token @TODO
+        $string = $this->_tokenSampler->visit($token->getAST());
```

Comment:
We used to name this kind of variable `$out` most of the time. This is variable that is going to be returned.